torch_xla/csrc/ops/unselect.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "torch_xla/csrc/ir.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+
+class Unselect : public Node {
```

Comment:
This name doesn't tell me much until I read the code. `GetSelectBase` would work better imo.

Reply:
Unselect as the inverse of Select. Like Undo is of Do :)
I'd leave it as is, unless we find a name which is clearly fitting better.